main.go

```diff
@@ -56,10 +56,29 @@ func run(ctx context.Context, args []string) error {
 		return fmt.Errorf("failed to chdir to %s: %w", workDir, err)
 	}
 
-	// Add task name to includes so rules can be filtered by task
+	// Build the complete CEL expression by combining user selector with task_name and resume filters
 	taskName := args[0]
-	includes["task_name"] = taskName
-	includes["resume"] = fmt.Sprint(resume)
+
+	// Build the base expression for task_name and resume
+	// The resume check should be: (!has(frontmatter.resume) || frontmatter.resume == expected)
+	// This allows tasks without a resume field to match, treating them as if they have resume=false
+	baseExpr := fmt.Sprintf("frontmatter.task_name == '%s' && (!has(frontmatter.resume) || frontmatter.resume == %v)",
+		taskName, resume)
+
+	// If user provided a selector, combine it with AND for task filtering
+	// User selectors CAN be used to distinguish between multiple tasks with the same name
+	var taskSelectorExpr string
```

Comment:
Make "includes.expression" "true" by default. Then consider how this simplifies things.